mmedit/datasets/pipelines/crop.py

```diff
@@ -84,6 +100,131 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class RandomResizedCrop(object):
+    """Crop data to random size and aspect ratio. Modified keys are
+       the attributes specified in "keys".
+
+    A crop of a random proportion of the original image
+    and a random aspect ratio of the original aspect ratio is made.
+    The cropped image is finally resized to a given size specified
+    by 'crop_size'.
+
+    This code is partially adopted from
+    torchvision.transforms.RandomResizedCrop:
+    [https://pytorch.org/vision/stable/_modules/torchvision/transforms/\
+        transforms.html#RandomResizedCrop].
+
+    Args:
+        keys (list[str]): The images to be resized and random-cropped.
+        crop_size (int | tuple[int]): Target spatial size (h, w).
+        scale (tuple[float], optional): Range of the proportion of the original
+                                        image to be cropped. Default:
+                                        (0.08, 1.0).
```

Comment:
We do not need to align texts. simply indenting the second and third lines is fine. Something like that. 
```
scale (tuple[float], optional): Range of the proportion of the original
  image to be cropped. Default: (0.08, 1.0).
```
Similar for others.